lessons/03 conditionals.py

```diff
@@ -46,17 +46,18 @@
     print('everything on the same indentation level belongs to this if-condition')
     print('so this will only get printed if the condition is met')
     print('and so does this')
-print('but not this!')
+print('but this gets printed regardless!')
 
 # %% [markdown]
-# - For indented blocks, we sometimes need `pass` as a placeholder. It lets us create an indented block, without defining any behavior for it yet.
+# - For indented blocks, we sometimes need `pass` as a placeholder. It lets us create an indented block, without defining any behavior for it yet. Everything after `pass` in the same block will be ignored.
 
 # %%
 earth = 'round'
 
 if earth == 'square':
     # TODO not sure yet how to handle this case
     pass
+    print('This will not be printed.')
```

Comment:
Pease add a second demonstration for this. This one was meant to illustrate a reasonable use case for `pass`, in which case you wouldn't add lines of code after it.

Reply:
(Also, does `pass` really cause the rest of the block to be skipped? In that case I learnt something new today.)